src/Calendar/CalendarFactory.php

```diff
@@ -32,7 +33,12 @@ public function createFromCdbCalendar(\CultureFeed_Cdb_Data_Calendar $cdbCalenda
                 $startDateString = $timestamp->getDate() . 'T00:00:00';
             }
         }
-        $startDate = !empty($startDateString) ? DateTimeFactory::dateTimeFromDateString($startDateString) : null;
+        $startDate = !empty($startDateString) ?
+            DateTimeFactory::fromFormat(
```

Comment:
Maybe we can hide some parameters and create a method called `DateTimeFactory::fromCdbFormat($startDateString)` and the format `'Y-m-d?H:i:s'` and timezone `new DateTimeZone('Europe/Brussels')` are reused inside that new method.

Reply:
This format `'Y-m-d?H:i:s'` only seems to make sense inside the Cdb namespace and it doesn't represent any PHP standard format.